docs/content/index.md

```diff
@@ -1,50 +1,41 @@
 # Get Started with DDEV
 
-[DDEV](https://github.com/drud/ddev) is an open source tool that makes it dead simple to get local PHP development environments up and running within minutes. It's powerful and flexible as a result of its per-project environment configurations, which can be extended, version controlled, and shared. In short, DDEV aims to allow development teams to use Docker in their workflow without the complexities of bespoke configuration.
+[DDEV](https://github.com/drud/ddev) is an open source tool for launching local PHP development environments in minutes.
 
-DDEV works great on macOS, Windows WSL2, traditional Windows, Linux and Gitpod.io. It works perfectly on amd64 and arm64 architectures, meaning it works fine natively on mac M1 systems and on Linux with both amd64 and arm64. It also works great on Gitpod, where you don't have to install anything at all.
+These environments can be extended, version controlled, and shared, so you can take advantage of a Docker workflow without Docker experience or bespoke configuration. Projects can be changed, powered down, or removed just as easily as they’re started.
 
 ## System Requirements
 
 === "macOS"
 
-    ### macOS
-    * DDEV runs natively on arm64 (Mac M1) systems as well as amd64 machines.
+    Runs natively on ARM64 (Apple Silicon) and AMD64 machines.
```

Comment:
Bah, bad call on my part. I struggled with this and should put a lot of these back until the tabs themselves can be linked.

At the moment it seems redundant to actively highlight a tab (indicating where you are) and then immediately see another heading that indicates the same thing. But the structural issue has to come before the content issue, so I can revert each of these heading removals.

Reply:
This is a problem with tabs in mkdocs-material. Amazing @mikesnoeren gave us a workaround in
* https://github.com/drud/ddev/pull/4046
* https://github.com/drud/ddev/pull/4084
* Others, https://github.com/drud/ddev/pulls?q=is%3Apr+is%3Aclosed+author%3Amikesnoeren

It was a super-serious problem in general (not allowing any link-based navigation) but wasn't completely solved by his work, so to get a predictable link you still need a header of some type. We almost had to completely revert the use of tabs because of it. 

It's important to know about this oddity and its offspring. I should have mentioned it early.